Wiki/Modules/FvwmButtons/index.md

```diff
@@ -41,4 +41,120 @@ you are going to need for the buttons.
+Say one wants to `swallow` an XClock and allocate 100% of the width and 80% of
+the height to the XClock. This can be defined as such:
+
+{% fvwm2rc %}
+*FvwmButtons: Rows 5
+*FvwmButtons: Columns 1
+*FvwmButtons: (1x4, Container)
+*FvwmButtons: (Swallow(UseOld, NoClose) "xclock" "Nop")
+*FvwmButtons: (End)
+{% endfvwm2rc %}
+
+Note that a `container` is created by defining a certain number of columns and
+rows and then using the keyword container. Elements inside the container are
+defined underneath this line and the container is then closed with the `End` command.
+
 ## Examples
+
+This advanced example is a complex set of elements forming a multitask FvwmButtons
+configuration. `Container` with command buttons, `Panel` that opens and closes one
+row of icon applications on a button press, and `Swallow` one `FvwmPager`.
+
+![Screenshot of FvwmButtons showing Container, Panel, and Swallow commands.](
+multitask-buttons.png){:.d-block .mx-auto .img-fluid}
+
+{% fvwm2rc %}
+## Multitask Panel buttons
+
+Colorset 10 fg #ffffff, bg #2b4e5e, hi #aaaaaa, sh #999999, Plain, NoShape
+ColorSet 15 RootTransparent Buffer, Tint #4a516a 60, fg white, bg black
+Colorset 16 VGradient 50 #638079 #b6b6b6, fg #ffffff
+Colorset 17 fg #ffffff, bg #028383, hi, sh, Plain, NoShape
+Colorset 18 fg #ffffff, bg #306ea2, hi #ffffff, sh #ffffff, Plain
+Colorset 19 fg #ffffff, bg #4b88a4, hi, sh, Plain, NoShape
+
+DestroyModuleConfig TaskButton: *
+*TaskButton: Geometry 160x240-0+0
+*TaskButton: Font "xft:Sans:size=10:antialias=True"
+*TaskButton: Colorset 16
+*TaskButton: Columns 2
+*TaskButton: Frame 0
+
+*TaskButton: (1x2, Panel(left, position module -10+4 top, steps 6, delay 10, indicator 48) \
+	"IconButton" 'Module FvwmButtons IconButton', Frame 2, Colorset 16)
+    
+*TaskButton: (1x2, Padding 0 0, Container(Padding 0 0, Frame 1, Columns 2))
+*TaskButton: (Icon $[icons]/move.png, Action Move)
+*TaskButton: (Icon $[icons]/curve.png, Action Restart)
+*TaskButton: (Icon $[icons]/resize.png, Action Resize)
+*TaskButton: (Icon $[icons]/close.png, Action Close)
+*TaskButton: (End)
+
+*TaskButton: (1x2, Title "1cal/3rofi", Icon $[icons]/calendar-rofi.png, \
+	Action(Mouse 1) "Exec exec script/Calendar.sh", \
+	Action(Mouse 3) 'Exec exec rofi -show', Frame 1)
+	
+*TaskButton: (1x2, Title "xclock", Swallow "xclock" "Exec exec xclock")
+*TaskButton: (2x2, Swallow "FvwmPagerTask" 'FvwmPager FvwmPagerTask * *')
+
+Style TaskButton NoTitle, NoHandles, !Title, WindowListSkip, Sticky, !Iconifiable, \
+BorderColorset 10, BorderWidth 0, StaysOnTop
+
+AddToFunc StartFunction I Module FvwmButtons TaskButtons
+{% endfvwm2rc %}
+
+{% fvwm2rc %}
+## Icon panel
+
+Colorset 7  fg #999999, bg #cbdcca, hi, sh, Plain, NoShape
+
+DestroyModuleConfig IconButton: *
+*IconButton: PressColorset 7
+*IconButton: Pixmap none
+*IconButton: Frame 0
+*IconButton: Columns 9  ## (as per numbers of buttons)
+
+*SimpleButton: (1x1, Icon icons/thunar.png, \
+            Action(Mouse 1) "Exec exec thunar")
```

Comment:
I'm not sure I understand this -- where is this configuration for SimpleButton defined?

Reply:
I updated the icon panel with /Wiki /Panels /SimpleButtons plus a few changes. This is a PR. The "Action Buttons" description still missing.

Reply:
It is one of the head titles on the FvwmButtons page:
https://www.fvwm.org/Wiki/Modules/FvwmButtons/